api/oplet/src/main/java/quarks/oplet/core/PeriodicSource.java

```diff
@@ -47,8 +48,8 @@ public void initialize(OpletContext<Void, T> context) {
     public synchronized void start() {
         ControlService cs = getOpletContext().getService(ControlService.class);
         if (cs != null)
-            cs.registerControl("periodic", getOpletContext().uniquify(getClass().getSimpleName()), 
-                    getAlias(), PeriodicMXBean.class, this);
+            cs.registerControl(TStream.TYPE, getOpletContext().uniquify(getClass().getSimpleName()), 
```

Comment:
Would it make sense to register the control against a class type rather than a String?  Instead of:

`<T> String registerControl(String type, String id, String alias, Class<T> controlInterface, T control);` where 
the `type` parameter identifies the type of the control MBean, use:

`<T> String registerControl(Class<?> type, String id, String alias, Class<T> controlInterface, T control);` where the `type` parameter is the type of object controlled by the MBean.